pkg/asset/manifests/vsphere/cloudproviderconfig.go

```diff
@@ -3,8 +3,8 @@ package vsphere
 import (
 	"bytes"
 	"fmt"
-
 	vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
+	"net"
```

Comment:
github won't let me make a suggestion due to the deleted line. It should look like this:
```go
"bytes"
"fmt"
"net"

vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
```